starfish/test/image/test_imagestack_cropped_load.py

```diff
@@ -37,22 +39,18 @@ def data(round_: int, ch: int, z: int) -> np.ndarray:
 
 
 def x_coordinates() -> Tuple[float, float]:
-    """Return the expected physical x coordinate value for a given round/ch tuple.  Note that in
-    real life, physical coordinates are not expected to vary for different ch values.  However, for
-    completeness of the tests, we are pretending they will."""
-    return 0.01, 0.01
+    """Return the expected physical x coordinates."""
```

Comment:
It feels like these could be hard-coded into the tests to make things simpler, or hard coded as constants into the tests, if the desire is to fix the values across the tests.